tools/report-converter/codechecker_report_converter/coccinelle/output_parser.py

```diff
@@ -39,21 +44,24 @@ def parse_message(self, it, line):
         Actual Parsing function for the given line
         """
         match = self.message_line_re.match(line)
+
+        checker_match = self.checker_name_re.match(line)
```

Comment:
I am just thinking to mention it under the coccinelle documentation (https://github.com/Ericsson/codechecker/blob/master/tools/report-converter/README.md#coccinelle) that the checker name will be the file name of the `.cocci` file or `coccinelle` by default.

Reply:
Yess will do that and squash the commit with the docs improvement commit :+1: